config/urls.py

```diff
@@ -24,4 +24,5 @@
     path('', TemplateView.as_view(template_name='home.html'), name='home'), # new
     path('accounts/signup/', TemplateView.as_view(template_name='signup.html'), name='signup'), # new
     path('maps/', include('maps.urls')),
+    path('parks/', include('parks.urls')),
```

Comment:
Not all the facilities are parks, I am wondering if this URL should be `facility/` instead? Let me know what you think!

Reply:
Since it's such a minor thing, can we deploy this for now and leave it as a nice to have?

Reply:
Done :)